src/main/java/seedu/address/model/event/Event.java

```diff
@@ -289,6 +299,22 @@ public void editPerson(Person personToEdit, Person editedPerson) {
             volunteers.add(editedPerson);
         }
     }
+
+    private void removeContactFromEventRoleOnEdit(Person personToEdit, Person editedPerson) {
```

Comment:
Perhaps in the future, we could decrease the repetitiveness of the code here, since there is quite a bit of code reuse!